src/python/grpcio/grpc/__init__.py

```diff
@@ -946,7 +979,9 @@ class Channel(abc.ABC):
     """
 
     @abc.abstractmethod
-    def subscribe(self, callback, try_to_connect=False):
+    def subscribe(self,
+                  callback: Callable[[ChannelConnectivity], None],
+                  try_to_connect: Optional[bool] = False) -> None:
```

Comment:
While the implemenation _does_ allow a `None` value for `try_to_connect`, I see no value in surfacing that here. Let's just make this `bool`.